iaso/models/org_unit.py

```diff
@@ -689,17 +689,17 @@ def save(self, *args, **kwargs):
         is_new = self.pk is None
         if is_new:
             # Save old values.
-            self.old_parent = self.org_unit.parent
+            self.old_parent_id = self.org_unit.parent_id
             self.old_name = self.org_unit.name
-            self.old_org_unit_type = self.org_unit.org_unit_type
+            self.old_org_unit_type_id = self.org_unit.org_unit_type_id
             self.old_location = self.org_unit.location
             self.old_opening_date = self.org_unit.opening_date
             self.old_closed_date = self.org_unit.closed_date
         super().save(*args, **kwargs)
         if is_new:
             # Wait for the instance to have an ID to save old m2m relations
-            self.old_groups.set(self.org_unit.groups.all())
-            self.old_reference_instances.set(self.org_unit.reference_instances.all())
+            self.old_groups.add(*self.org_unit.groups.all())
```

Comment:
I'm curious, why is this useful?

Reply:
Because using `add()` will not call any `save()` methods, but rather create the relationships using `QuerySet.bulk_create()`.

I wasn't aware of this, see https://docs.djangoproject.com/en/dev/ref/models/relations/#django.db.models.fields.related.RelatedManager.add

Reply:
Ooooh, nifty

Reply:
I'd be curious to see the change in the generated SQL, but we can have that discussion outside of github :-)